flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -63,6 +73,14 @@ class FlxGamepad implements IFlxDestroyable
 	 * Helper class to get the justMoved, justReleased, and float values of analog input.
 	 */
 	public var analog(default, null):FlxGamepadAnalogList;
+	/**
+	 * Helper class to get the float values of motion-sensing input, if it is available
+	 */
+	public var motion(default, null):FlxGamepadMotionValueList;
+	/**
+	 * Helper class to get the float values of mouse-like pointer nput, if it is available
+	 */
+	public var pointer(default, null):FlxGamepadPointerValueList;
```

Comment:
Hmmm, this will take some thinking.

So, we don't have the consoles hooked up yet, but here's every device that might use the "gamepad.pointer" API:

- Nintendo 3DS
- Nintendo WiiU
- OUYA touchpad on controller
- PS4 touchpad on controller
- Steam controller (multiple touchpads)
- Wii remote (IR camera)

So the obvious super easy thing to do on the native console side is to rig up the input from the touchscreen to Lime/OpenFL's mouse/touch events, report absolute world coordinates, and call it a day. Flixel doesn't even need to "know" about this.

The problem with this solution is that you might have more than one device that has this kind of "pointery" input, and the mouse/touch API is built around the paradigm that there is ONE mouse/touch surface on the device/computer. So if we do that with the API, then you can't make a PS4 game where everyone is using their own touchpad.

On the 3DS obviously we should just use the same touch API we use for mobile. On the WiiU -- can you have more than one gamepad active on the same console?

So there's this issue to work out.

Other than that, what do you propose besides "gamead.pointer" as the name?